attachment_s3/__init__.py

```diff
@@ -1,2 +1,10 @@
-
 from . import models
+from . import controllers
+
+from odoo import api, SUPERUSER_ID
+
+
+def _auto_create_bucket(cr, registry):
+    # create the S3 bucket after module installed
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    env["ir.attachment"]._get_s3_bucket()
```

Comment:
@hiren-serpentcs You also need to set the config parameter and force the storage like here:
https://github.com/ursais/odoo-template/blob/14.0/odoo/songs/setup/base.py#L51